ports/servoshell/webview.rs

```diff
@@ -112,7 +113,7 @@ where
 
     pub fn handle_window_events(&mut self, events: Vec<EmbedderEvent>) {
         for event in events {
-            trace!("embedder <- window EmbedderEvent {:?}", event);
+            trace_to_servo!(event, "{event:?}");
```

Comment:
"trace_embedder_event" perhaps?

Reply:
done in 198decce710d39b80ab8e02b35396515145186b5